chromium_src/components/os_crypt/os_crypt_win.cc

```diff
@@ -1,8 +1,16 @@
 #include "base/command_line.h"
+#include "brave/common/brave_switches.h"
+
+// switches::kDisableEncryption
+const char kDisableEncryption[] = "disable-encryption";
 
 namespace {
-bool IsEncryptionDisabled() {
-  return base::CommandLine::ForCurrentProcess()->HasSwitch("disable-encryption");
+bool IsEncryptionDisabled(std::string text1, std::string* text2) {
+  if (base::CommandLine::ForCurrentProcess()->HasSwitch(kDisableEncryption)) {
+    text2 = &text1;

```

Comment:
This isn't safe because you're pointing text2 to the address of a temp var. This should be *text2 = text1 and pass `const std::string text1`. Also I understand that you can't use ciphertext/plaintext var names here because they are swapped in the calls so maybe `IsEncryptionDisabled (const std::string input_text, std::string& output_text)`?

Reply:
Yes this was already done from this [commit](https://github.com/brave/brave-core/pull/795/commits/1458f7b7915e75ee28115f5949caa76c4fa5370e) :)